_data/payments.yml

```diff
@@ -61,7 +61,7 @@ websites:
       exceptions:
          text: "Software 2FA is only available for Swedish citizens with BankID."
 
-    - name: Litle & Co
+    - name: "Litle & Co"
```

Comment:
Reason for change:

http://stackoverflow.com/questions/19109912/do-i-need-quotes-for-strings-in-yaml